app/javascript/dashboard/components-next/combobox/ComboBoxDropdown.vue

```diff
@@ -46,6 +46,11 @@ const isSelected = option => {
   return option.value === props.selectedValues;
 };
 
+const onInputSearch = event => {
+  emit('update:searchValue', event.target.value);
```

Comment:
We can use `defineModel` and avoid manually emitting this. We can keep the `emit('search', event.target.value);` anyway